src/workspace-element.coffee

```diff
@@ -95,6 +95,50 @@ class WorkspaceElement extends HTMLElement
 
   focusPaneViewOnRight: -> @paneContainer.focusPaneViewOnRight()
 
+  copyItemToPaneOnLeft: (destroyOriginal) ->
```

Comment:
Same thought on the method name... maybe slightly more explicit at the expense of brevity: `copyActiveItemToPaneOnLeft`